r10k.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'log4r',     '1.1.10'
   s.add_dependency 'multi_json', '~> 1.10'
 
-  s.add_dependency 'puppet_forge', '~> 2.3.0'
+  s.add_dependency 'puppet_forge', '~> 3.1.0'
```

Comment:
I wonder if using `>= 2.3.0` would work? That way anyone still using Ruby 2.3 could keep using older versions of `puppet_forge`.

Reply:
Oh sure, that would work just fine. I can update that.